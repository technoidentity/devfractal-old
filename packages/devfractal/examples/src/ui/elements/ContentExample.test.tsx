import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Content } from './ContentExample'

test('Content Example', () => {
  const component: ReactTestRenderer = renderer.create(<Content />)
  expect(component.toJSON()).toMatchSnapshot()
})
