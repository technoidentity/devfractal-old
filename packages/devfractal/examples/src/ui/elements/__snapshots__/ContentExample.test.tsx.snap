// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Example
    </h1>
    <div
      className="content"
    >
      <div
        className="box"
      >
        <ol
          className="is-lower-alpha"
        >
          <li>
            Coffee
          </li>
          <li>
            Tea
          </li>
          <li>
            Milk
          </li>
        </ol>
        <ol
          className="is-lower-roman"
        >
          <li>
            Coffee
          </li>
          <li>
            Tea
          </li>
          <li>
            Milk
          </li>
        </ol>
        <ol
          className="is-upper-alpha"
        >
          <li>
            Coffee
          </li>
          <li>
            Tea
          </li>
          <li>
            Milk
          </li>
        </ol>
        <ol
          className="is-upper-roman"
        >
          <li>
            Coffee
          </li>
          <li>
            Tea
          </li>
          <li>
            Milk
          </li>
        </ol>
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Sizes
    </h1>
    <div
      className="has-text-weight-bold"
    >
      Medium
    </div>
    <div
      className="box"
    >
      <div
        className="is-size-2 has-text-weight-bold"
      >
        Hello World
      </div>
      <div
        className="content is-medium"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla accumsan, metus ultrices eleifend gravida, nulla nunc varius lectus, nec rutrum justo nibh eu lectus. Ut vulputate semper dui. Fusce erat odio, sollicitudin vel erat vel, interdum mattis neque.
      </div>
      <p
        className="is-size-3 has-text-weight-bold"
      >
        Second level
      </p>
      <div
        className="content is-medium"
      >
        Curabitur accumsan turpis pharetra augue tincidunt blandit. Quisque condimentum maximus mi, sit amet commodo arcu rutrum id. Proin pretium urna vel cursus venenatis. Suspendisse potenti. Etiam mattis sem rhoncus lacus dapibus facilisis. Donec at dignissim dui. Ut et neque nisl.
        <ol
          className=""
        >
          <li>
            In fermentum leo eu lectus mollis, quis dictum mi aliquet.
          </li>
          <li>
            Morbi eu nulla lobortis, lobortis est in, fringilla felis.
          </li>
          <li>
            Aliquam nec felis in sapien venenatis viverra fermentum nec lectus.
          </li>
          <li>
            Ut non enim metus
          </li>
        </ol>
      </div>
    </div>
    <div
      className="has-text-weight-bold"
    >
      Large
    </div>
    <div
      className="box"
    >
      <div
        className="is-size-2 has-text-weight-bold"
      >
        Hello World
      </div>
      <div
        className="content is-large"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla accumsan, metus ultrices eleifend gravida, nulla nunc varius lectus, nec rutrum justo nibh eu lectus. Ut vulputate semper dui. Fusce erat odio, sollicitudin vel erat vel, interdum mattis neque.
      </div>
      <p
        className="is-size-3 has-text-weight-bold"
      >
        Second level
      </p>
      <div
        className="content is-large"
      >
        Curabitur accumsan turpis pharetra augue tincidunt blandit. Quisque condimentum maximus mi, sit amet commodo arcu rutrum id. Proin pretium urna vel cursus venenatis. Suspendisse potenti. Etiam mattis sem rhoncus lacus dapibus facilisis. Donec at dignissim dui. Ut et neque nisl.
        <ol
          className=""
        >
          <li>
            In fermentum leo eu lectus mollis, quis dictum mi aliquet.
          </li>
          <li>
            Morbi eu nulla lobortis, lobortis est in, fringilla felis.
          </li>
          <li>
            Aliquam nec felis in sapien venenatis viverra fermentum nec lectus.
          </li>
          <li>
            Ut non enim metus
          </li>
        </ol>
      </div>
    </div>
    <div
      className="has-text-weight-bold"
    >
      Small
    </div>
    <div
      className="box"
    >
      <div
        className="is-size-2 has-text-weight-bold"
      >
        Hello World
      </div>
      <div
        className="content is-small"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla accumsan, metus ultrices eleifend gravida, nulla nunc varius lectus, nec rutrum justo nibh eu lectus. Ut vulputate semper dui. Fusce erat odio, sollicitudin vel erat vel, interdum mattis neque.
      </div>
      <p
        className="is-size-3 has-text-weight-bold"
      >
        Second level
      </p>
      <div
        className="content is-small"
      >
        Curabitur accumsan turpis pharetra augue tincidunt blandit. Quisque condimentum maximus mi, sit amet commodo arcu rutrum id. Proin pretium urna vel cursus venenatis. Suspendisse potenti. Etiam mattis sem rhoncus lacus dapibus facilisis. Donec at dignissim dui. Ut et neque nisl.
        <ol
          className=""
        >
          <li>
            In fermentum leo eu lectus mollis, quis dictum mi aliquet.
          </li>
          <li>
            Morbi eu nulla lobortis, lobortis est in, fringilla felis.
          </li>
          <li>
            Aliquam nec felis in sapien venenatis viverra fermentum nec lectus.
          </li>
          <li>
            Ut non enim metus
          </li>
        </ol>
      </div>
    </div>
  </section>,
]
`;
